docs/README.md

```diff
@@ -0,0 +1,33 @@
+# Website
+
+This website is built using [Docusaurus 2](https://docusaurus.io/), a modern static website generator.
+
+### Installation
+
+```
+$ yarn
+```
+
+### Local Development
+
+```
+$ yarn start
+```
+
+This command starts a local development server and opens up a browser window. Most changes are reflected live without having to restart the server.
+
+### Build
+
+```
+$ yarn build
+```
+
+This command generates static content into the `build` directory and can be served using any static contents hosting service.
+
+### Deployment
+
+```
+$ GIT_USER=<Your GitHub username> USE_SSH=true yarn deploy
```

Comment:
I would use a seperate repo: https://github.com/woodpecker-ci/pages

Reply:
Docusaurus uses its config `organizationName: 'woodpecker-ci', projectName: 'woodpecker-ci.github.io',` to push to a repo called `woodpecker-ci.github.io` in the `woodpecker-ci` org. [See docs](https://docusaurus.io/docs/deployment#docusaurusconfigjs-settings)

Reply:
INFO to all reviewers: changed repo to https://github.com/woodpecker-ci/woodpecker-ci.github.io ;)